071-GitHubCopilot/Coach/Solution-00.md

```diff
@@ -0,0 +1,12 @@
+# Challenge 00 - <Title of Challenge> - Coach's Guide 
```

Comment:
what should this title be?

Reply:
Updated to include title: Prerequisites - Ready, Set, GO! - Coach's Guide